misk/src/main/kotlin/misk/eventrouter/EventRouterConnectionAction.kt

```diff
@@ -0,0 +1,18 @@
+package misk.eventrouter
+
+import misk.web.ConnectWebSocket
+import misk.web.actions.WebAction
+import misk.web.actions.WebSocket
+import misk.web.actions.WebSocketListener
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+internal class EventRouterConnectionAction : WebAction {
+  @Inject lateinit var realEventRouter: RealEventRouter
+
+  @ConnectWebSocket("/eventrouter")
+  fun eventRouter(webSocket: WebSocket): WebSocketListener {
```

Comment:
interesting that we don’t need a handle to the websocket

Reply:
well it's passed into the websocket listener anyway right but yeah